combat-trainer.lic

```diff
@@ -4391,7 +4395,10 @@ class GameState
   # If the main hand weapon skill is doublestrikable then `determine_doublestrike_skill`
   # is used to determine what to equip offhand to perform the maneuver.
   def doublestrikable_weapon_skill?
-    @doublestrike_trainables.include?(weapon_skill) && (melee_weapon_skill? || thrown_skill?) && !twohanded_weapon_skill?
+    @doublestrike_trainables.include?(weapon_skill) && # mainhand weapon is listed for doublestrikes
```

Comment:
Just a style comment - 

I understand why this is all chained together, but personally think it might be cleaner (easier to read) if it was a series of early outs, i.e.
```
  return unless @doublestrike_trainables.include?(weapon_skill)
  return unless melee_weapon_skill? || thrown_skill?
  ...
```

Curious what other people think, and what is more 'Ruby' correct.